python/test/test_supporting_elements.py

```diff
@@ -15,4 +15,11 @@ def stability_map(d_in):
         stability_map=stability_map
     )
     
-    assert trans.map((3, 4, 3)) == (3, 4, 3)
\ No newline at end of file
+    assert trans.map((3, 4, 3)) == (3, 4, 3)
+
+
+def test_atom_domain():
+    # Checks that all primitive types are construct-able over FFI.
+    # This ensures that all Polars dtypes can be used in debug builds.
+    for ty in dp.PRIMITIVE_TYPES:
+        dp.option_domain(dp.atom_domain(T=ty))
```

Comment:
```suggestion
import pytest
@pytest.mark.parametrize("ty", dp.PRIMITIVE_TYPES)
def test_atom_domain(ty):
    # Checks that all primitive types are construct-able over FFI.
    # This ensures that all Polars dtypes can be used in debug builds.
    dp.option_domain(dp.atom_domain(T=ty))
```
(Or move the import to the top of file: Just included here for completeness.)

- Better reporting, if running with verbose
- All types are run, instead of failing on first